examples/kafka/kafka-ephemeral-oauth-single-keycloak-authz.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.7.0
+    replicas: 1
+    listeners:
+      - name: plain
+        port: 9092
+        type: internal
+        tls: false
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+        authentication:
+          type: oauth
+          validIssuerUri: https://${SSO_HOST}/auth/realms/kafka-authz
+          jwksEndpointUri: https://${SSO_HOST}/auth/realms/kafka-authz/protocol/openid-connect/certs
+          userNameClaim: preferred_username
+          maxSecondsWithoutReauthentication: 3600
+          tlsTrustedCertificates:
+            - secretName: oauth-server-cert
+              certificate: sso.crt
+    authorization:
+      type: keycloak
+      clientId: kafka
+      tokenEndpointUri: https://${SSO_HOST}/auth/realms/kafka-authz/protocol/openid-connect/token
+      tlsTrustedCertificates:
+        - secretName: oauth-server-cert
+          certificate: sso.crt
+      delegateToKafkaAcls: true
+      superUsers:
+        - User:service-account-kafka
+    logging:
+      type: inline
+      loggers:
+        log4j.logger.io.strimzi: "TRACE"
+        log4j.logger.kafka: "DEBUG"
+        log4j.logger.org.apache.kafka: "DEBUG"
```

Comment:
Why do we need this section?

Reply:
For the authz examples where I direct the user to check the Kafka broker log to get a better understanding of what's happening.